src/main/java/com/minecolonies/colony/WorkManager.java

```diff
@@ -1,26 +1,28 @@
 package com.minecolonies.colony;
 
+import com.minecolonies.MineColonies;
 import com.minecolonies.colony.workorders.AbstractWorkOrder;
+import com.minecolonies.network.messages.ColonyViewRemoveCitizenMessage;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.minecolonies.network.messages.ColonyViewRemoveCitizenMessage'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)